docs/content/quickstart.md

```diff
@@ -22,12 +22,12 @@ For Spin, follow [the Spin quickstart guide](https://spin.fermyon.dev/quickstart
 
 For the `bart` CLI, there are two options:
 - download the latest `bart` binary [release](https://github.com/fermyon/bartholomew/releases/), or
-- clone and install `bart` from source, using the following commands:
+- clone and build `bart` from source, using the following commands (NOTE: `bart` will need to be installed to a user executable directory such as `/usr/bin/` to be run when built this way.):
```

Comment:
Thanks for the PR! An addition of where the binary can be found would be great (i.e) `target/release/bart`

Reply:
done in 70d7c59691b1a885dd46c30952ced69f40a5cdf1